src/olympia/blocklist/models.py

```diff
@@ -261,12 +264,18 @@ def __str__(self):
         trimmed = [rep if len(rep) < 40 else rep[0:37] + '...' for rep in repr]
         return f'{self.get_signoff_state_display()}: {"; ".join(trimmed)}'
 
+    @property
+    def in_legacy_blocklist(self):
+        # This is for consitency with Block.  Should be a boolean anyway.
```

Comment:
Why is there two spaces between the two sentences? Also, shouldn't this comment be added to the same method above (in the other class)?

Reply:
> Why is there two spaces between the two sentences? 

https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/ 😝 (In other words, I was taught to type in such a bygone era that consideration of typewriter mechanics was still a possible thing)

> Also, shouldn't this comment be added to the same method above (in the other class)?

It's not a boolean in `Block` so it's an actually useful function.  So I'd lean "no" ... but maybe?